### PR TITLE
More stories and some fixes

### PR DESCRIPTION
**Pull request by Seggan**
Added more stories and did some grammatical fixes in yours

### CODE REVIEW

File: src/main/resources/blocks.yml [hunk 1]
```diff
@@ -9794,13 +9794,18 @@ POWDER_SNOW_BUCKET:
 POWERED_RAIL:
   tier: 1
   elements:
-    -
+    - MECHANICAL
+    - HUMAN
   story:
-    name:
-    type:
+    name: Level up
```
Comment (Sefiraat) [inline src/main/resources/blocks.yml:9800]:
```suggestion
    name: Level Up
```

File: src/main/resources/blocks.yml [hunk 2]
```diff
@@ -9794,13 +9794,18 @@ POWDER_SNOW_BUCKET:
...
+    type: MECHANICAL
     lore:
-      -
+      - When Dr. Minestein accidentally dropped
+      - some redstone on the railroad, he suddenly
+      - discovered the next level in affordable
+      - transportation
     shards:
```
Comment (Sefiraat) [inline src/main/resources/blocks.yml:9807]:
```suggestion
    shards: [ 0,1,0,1,1,0,0,0,0 ]
```

File: src/main/resources/blocks.yml [hunk 3]
```diff
@@ -9794,13 +9794,18 @@ POWDER_SNOW_BUCKET:
 POWERED_RAIL:
   tier: 1
```
Comment (Sefiraat) [inline src/main/resources/blocks.yml:9795]:
```suggestion
  tier: 2
```

File: src/main/resources/blocks.yml [hunk 4]
```diff
@@ -11178,13 +11186,17 @@ SMALL_DRIPLEAF:
 SMITHING_TABLE:
   tier: 1
```
Comment (Sefiraat) [inline src/main/resources/blocks.yml:11187]:
```suggestion
  tier: 2
```

File: src/main/resources/blocks.yml [hunk 5]
```diff
@@ -11178,13 +11186,17 @@ SMALL_DRIPLEAF:
...
-    name:
-    type:
+    name: Gilded
+    type: HUMAN
     lore:
-      -
+      - Somehow, the magic of this table manages
+      - to bend and form the hardest known material
+      - around your favorite diamond tools
     shards:
```
Comment (Sefiraat) [inline src/main/resources/blocks.yml:11198]:
```suggestion
    shards: [ 0,1,0,0,2,0,0,0,0 ]
```

File: src/main/resources/blocks.yml [hunk 6]
```diff
@@ -11358,13 +11370,18 @@ SMOOTH_STONE_SLAB:
...
-    type:
+    name: Crystallized
+    type: ELEMENTAL
     lore:
-      -
+      - Made from the Crystamae in
+      - the air, this wonderful material
+      - is only found in the coldest
+      - of locations
     shards:
```
Comment (Sefiraat) [inline src/main/resources/blocks.yml:11383]:
```suggestion
    shards: [ 1,0,0,0,0,0,1,0,0 ]
```

File: src/main/resources/blocks.yml [hunk 7]
```diff
@@ -11463,13 +11480,17 @@ SPAWNER:
 SPECTRAL_ARROW:
   tier: 1
```
Comment (Sefiraat) [inline src/main/resources/blocks.yml:11481]:
```suggestion
  tier: 2
```

File: src/main/resources/blocks.yml [hunk 8]
```diff
@@ -11463,13 +11480,17 @@ SPAWNER:
...
-    name:
-    type:
+    name: Sparkling
+    type: HUMAN
     lore:
-      -
+      - Covered with precious glowstone
+      - dust, this magical arrow is hepful
+      - for spotting your enemies
     shards:
```
Comment (Sefiraat) [inline src/main/resources/blocks.yml:11492]:
```suggestion
    shards: [ 1,0,0,0,1,0,1,0,0 ]
```

File: src/main/resources/blocks.yml [hunk 9]
```diff
@@ -12240,24 +12261,28 @@ SWEET_BERRIES:
...
   story:
-    name:
-    type:
+    name: Barrier
+    type: ELEMENTAL
     lore:
-      -
+      - What could be more annoying
+      - than tall grass?
     shards:
```
Comment (Sefiraat) [inline src/main/resources/blocks.yml:12271]:
```suggestion
    shards: [ 1,0,0,0,0,0,0,1,0 ]
```

File: src/main/resources/blocks.yml [hunk 10]
```diff
@@ -12240,24 +12261,28 @@ SWEET_BERRIES:
...
+    author: Seggan
 
 TARGET:
   tier: 1
```
Comment (Sefiraat) [inline src/main/resources/blocks.yml:12275]:
```suggestion
  tier: 2
```

File: src/main/resources/blocks.yml [hunk 11]
```diff
@@ -12240,24 +12261,28 @@ SWEET_BERRIES:
...
+    - MECHANICAL
   story:
-    name:
-    type:
+    name: Robin Hood
+    type: HUMAN
     lore:
-      -
+      - Are you better than Robin Hood?
     shards:
```
Comment (Sefiraat) [inline src/main/resources/blocks.yml:12284]:
```suggestion
    shards: [ 0,2,0,0,1,0,0,0,0 ]
```

File: src/main/resources/blocks.yml [hunk 12]
```diff
@@ -12276,13 +12301,17 @@ TERRACOTTA:
 TINTED_GLASS:
   tier: 1
```
Comment (Sefiraat) [inline src/main/resources/blocks.yml:12302]:
```suggestion
  tier: 2
```

File: src/main/resources/blocks.yml [hunk 13]
```diff
@@ -12276,13 +12301,17 @@ TERRACOTTA:
...
-    name:
-    type:
+    name: Darkness
+    type: VOID
     lore:
-      -
+      - Corrupted with impure Crystamae,
+      - this glass is forever destined
+      - to fail its purpose
     shards:
```
Comment (Sefiraat) [inline src/main/resources/blocks.yml:12313]:
```suggestion
    shards: [ 1,0,0,0,1,0,0,2,0 ]
```

File: src/main/resources/blocks.yml [hunk 14]
```diff
@@ -12371,13 +12400,17 @@ TRIDENT:
...
-    name:
-    type:
+    name: Tripping up
+    type: MECHANCIAL
     lore:
-      -
+      - When Dr. Minestein invented the tripwire
+      - hook, he tripped, and out came the idea
+      - of quantum minechanics
     shards:
```
Comment (Sefiraat) [inline src/main/resources/blocks.yml:12412]:
```suggestion
    shards: [ 0,1,0,0,1,0,0,0,1 ]
```

File: src/main/resources/blocks.yml [hunk 15]
```diff
@@ -12953,13 +12986,17 @@ WAXED_WEATHERED_CUT_COPPER_STAIRS:
 WEATHERED_COPPER:
   tier: 1
```
Comment (Sefiraat) [inline src/main/resources/blocks.yml:12987]:
```suggestion
  tier: 3
```

File: src/main/resources/blocks.yml [hunk 16]
```diff
@@ -12953,13 +12986,17 @@ WAXED_WEATHERED_CUT_COPPER_STAIRS:
...
-    name:
-    type:
+    name: Rusted to perfection
+    type: ELEMENTAL
     lore:
-      -
+      - The most beautiful form of copper, this
+      - block has gone through many trials to reach
+      - this level of perfection
     shards:
```
Comment (Sefiraat) [inline src/main/resources/blocks.yml:12998]:
```suggestion
    shards: [ 2,1,0,0,1,0,0,0,0 ]
```

File: src/main/resources/blocks.yml [hunk 17]
```diff
@@ -13247,13 +13284,17 @@ WITHER_ROSE:
 WITHER_SKELETON_SKULL:
   tier: 1
```
Comment (Sefiraat) [inline src/main/resources/blocks.yml:13285]:
```suggestion
  tier: 3
```

File: src/main/resources/blocks.yml [hunk 18]
```diff
@@ -13247,13 +13284,17 @@ WITHER_ROSE:
...
-    name:
-    type:
+    name: Spooky scary skele...
+    type: VOID
     lore:
-      -
+      - Unfortunately, I cannot use the
+      - full title due to advancement
+      - copyright laws
     shards:
```
Comment (Sefiraat) [inline src/main/resources/blocks.yml:13296]:
```suggestion
    shards: [ 0,0,0,1,0,1,0,2,0 ]
```

File: src/main/resources/blocks.yml [hunk 19]
```diff
@@ -13339,13 +13380,16 @@ WRITABLE_BOOK:
 WRITTEN_BOOK:
   tier: 1
   elements:
-    -
+    - PHILOSOPHICAL
+    - HUMAN
   story:
-    name:
-    type:
+    name: The press
```
Comment (Sefiraat) [inline src/main/resources/blocks.yml:13386]: The Press

File: src/main/resources/blocks.yml [hunk 20]
```diff
@@ -13339,13 +13380,16 @@ WRITABLE_BOOK:
...
+    type: PHILOSOPHICAL
     lore:
-      -
+      - Oh the day when humans invented
+      - writing!
     shards:
```
Comment (Sefiraat) [inline src/main/resources/blocks.yml:13391]:
```suggestion
    shards: [ 0,0,0,1,1,0,0,0,1 ]
```

**Review by Sefiraat — CHANGES_REQUESTED** [review]

**Comment by Sefiraat** [comment]
Epic - tyvm, need to look up this Minestein feller xD

**Comment by Seggan** [comment]
Minestein is a mixture of Minecraft and Einstein, just a word i made up :)

**Review by Sefiraat — APPROVED** [review]

**Pull request merged**